tests/testJobManager.py

```
@@ -0,0 +1,142 @@
+        self.assertTrue(self.jobManager.running)
+
+    def test__getNextID(self):
+        init_id = self.jobManager.nextId
+        for i in range(1, Config.MAX_JOBID + 100):
+            id = self.jobManager._getNextID()
+            self.assertEqual(init_id + i - 1, id)
+        self.jobManager.nextId = init_id
+
+    def test_manage_worker_works(self):
+        def mock_getNextPendingJob():
+            time.sleep(0.2)
```

Ruff

`time` may be undefined, or defined from star imports

(F405)